corehq/apps/hqadmin/management/commands/clone_domain.py

```diff
@@ -282,15 +265,6 @@ def copy_ucr_datasources(self):
 
             old_id, new_id = self.save_couch_copy(datasource, self.new_domain)
             datasource_map[old_id] = new_id
-        for static_datasource in StaticDataSourceConfiguration.by_domain(self.existing_domain):
-            table_id = static_datasource.get_id.replace(
-                StaticDataSourceConfiguration._datasource_id_prefix + self.existing_domain + '-',
-                ''
-            )
-            new_id = StaticDataSourceConfiguration.get_doc_id(self.new_domain, table_id)
-            # check that new datasource is in new domain's list of static datasources
-            StaticDataSourceConfiguration.by_id(new_id)
-            datasource_map[static_datasource.get_id] = new_id
```

Comment:
It appeared to do nothing. The `datasource_map` was never used in the static reports part of `copy_ucrs` which is the only time that map is used at all. I had assumed this is because the datasources on the config dont change for static reports. It now looks to me like the domain check is relevant so maybe only the last line should be deleted?